example/example-integration.js

```diff
@@ -92,7 +92,7 @@
     // initialize the ads plugin, passing in any relevant options
     player.ads(options);
 
-    // request ad inventory whenever the player gets new content to play
+    // request ad inventory whenever the player gets content to play
```

Comment:
maybe this listener should be updated to listen to contentchanged?

Reply:
Originally I didn't change it because it seems to run on the initial contentupdate. Now that I look at it again it seems like it might request ads twice sometimes as it is? I am going to update it.